decidim-core/lib/decidim/exporters/exporter.rb

```diff
@@ -10,7 +10,7 @@ class Exporter
       #
       # collection - An Array with the collection to be exported.
       # serializer - A Serializer to be used during the export.
-      def initialize(collection, serializer)
+      def initialize(collection, serializer = Serializer)
```

Comment:
What's this `Serializer?`? Is this something from our code?

Reply:
Yeah, it's a passthrough serializer, returns the object itself.